api-server/src/server/mod.rs

```diff
@@ -21,27 +21,25 @@ pub use crate::context;
 
 type ServiceFuture = BoxFuture<'static, Result<Response<Body>, crate::ServiceError>>;
 
-use crate::{Api, CeramicEventsPostResponse, CeramicSubscribeSortKeySortValueGetResponse};
+use crate::{Api, CeramicEventsPostResponse, CeramicSubscribeSortkeySortvalueGetResponse};
 
 mod paths {
     use lazy_static::lazy_static;
 
     lazy_static! {
         pub static ref GLOBAL_REGEX_SET: regex::RegexSet = regex::RegexSet::new(vec![
             r"^/ceramic/events$",
-            r"^/ceramic/subscribe/(?P<sort-key>[^/?#]*)/(?P<sort-value>[^/?#]*)$"
+            r"^/ceramic/subscribe/(?P<sortkey>[^/?#]*)/(?P<sortvalue>[^/?#]*)$"
```

Comment:
"error: invalid capture group character" thrown before, removed dash

Reply:
Yeah I ran into that yesterday too, thanks for the fix.